introduction.md

```diff
@@ -4,3 +4,9 @@ Introduction
 My name is Attila, but on GitHub I'm known as atifar. Looking forward to picking up a lot of useful knowledge about using GitHub effectively in this course.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?